lib/avatar/getCurrentAvatar.js

```diff
@@ -1,7 +1,20 @@
+/* @module Avatar */
```

Comment:
Is this meant to be here? I note none of the other avatar-related files have a `@module` decorator.